spinny_gun.py

```diff
@@ -80,6 +80,53 @@ def intersects(rect, radius, center):
     return corner_distance_sq <= radius ** 2.0
 
 
+def gameOver():
+    """Game over screen function"""
+
+    pygame.mixer.music.pause()
+
+    gameOver_surf_1, gameOver_rect_1 = text_objects(
+        "GAME OVER",
+        MEDIUM_TEXT,
+        WHITE,
+        (DISPLAY_WIDTH * 0.5, DISPLAY_HEIGHT * 0.375),
+    )
+
+    gameOver_surf_2, gameOver_rect_2 = text_objects(
+        "Press 'P' to return to menu",
```

Comment:
```suggestion
        "Press 'p' to return to menu",
```
Since we're using a lowercase p, that should be displayed I think